.github/workflows/ci.yml

```diff
@@ -2,14 +2,21 @@ name: CI
 
 on: [push]
 
-env:
-  CI: true
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 jobs:
+  lint:
+    uses: inrupt/typescript-sdk-tools/.github/workflows/reusable-lint.yml@v1.4.2
+
   build:
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
```

Comment:
Are we setting this flag up for unit tests too?

Reply:
That's because this is running the unit tests in a matrix across node.js 16 and 14, not just the E2E tests.

Reply:
If we include 18, then we'd have 18 as experimental

Reply:
Well I'm not sure: when we move to supporting Node 18, is it going to be considered experimental? In other words, are any of our unit tests experimental at some point, or does it only apply to e2e tests, where the remote environment is experimental rather than our runtime?

Reply:
Short answer is: I don't know.